sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # the stims are the same for each trial?
</pre><p>This seems overly complicated. You already have these variables in their right form, EXCEPT that you will want to build a new design matrix for the stimulus using FFstimR. Use a time-embedding function that takes any stim (FFstim or FFstimR) and makes the time-embedded version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>